src/Management/src/EndpointBase/Metrics/Observer/EventCounterListener.cs

```diff
@@ -21,17 +21,25 @@ public class EventCounterListener : EventListener
     private readonly string _eventSourceName = "System.Runtime";
     private readonly string _eventName = "EventCounters";
     private readonly IMetricsObserverOptions _options;
+    private readonly bool _isInitialized = false;
+
+    private readonly Dictionary<string, string> _refreshInterval = new () { { "EventCounterIntervalSec", "1" } }; // TODO: Make it configurable
```

Comment:
Should this TODO be addressed in this PR? If not, can you remove it from the code and create an issue for it?

Reply:
I think its outside the scope of this change - Ill create an issue.

Reply:
Can we resolve this conversation?